services/wafv2.py

```diff
@@ -2,28 +2,83 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("wafv2", region_name="us-east-1")
+
+cloudfront_client = boto3.client("cloudfront", "us-east-1")
 
 
 def wafv2_logging_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    webacls = client.list_web_acls(Scope="REGIONAL")["WebACLs"]
+
+    for webacl in webacls:
+        print(webacl["ARN"])
+        configuration = client.get_logging_configuration(ResourceArn=webacl["ARN"])
+        if configuration["LoggingConfiguration"] != []:
+            compliant_resources.append(webacl["ARN"])
+        else:
+            non_compliant_resources.append(webacl["ARN"])
```

Comment:
CloudFront용 WebACL 추가
```suggestion
    regional_web_acls = client.list_web_acls(Scope="REGIONAL")["WebACLs"]
    cloudfront_web_acls = global_client.list_web_acls(Scope="CLOUDFRONT")["WebACLs"]

    for web_acl in regional_web_acls:
        try:
            configuration = client.get_logging_configuration(ResourceArn=web_acl["ARN"])
            compliant_resources.append(web_acl["ARN"])
        except Exception as e:
            if e.__class__.__name__ == "WAFNonexistentItemException":
                non_compliant_resources.append(web_acl["ARN"])
            else:
                raise e

    for web_acl in cloudfront_web_acls:
        try:
            configuration = global_client.get_logging_configuration(ResourceArn=web_acl["ARN"])
            compliant_resources.append(web_acl["ARN"])
        except Exception as e:
            if e.__class__.__name__ == "WAFNonexistentItemException":
                non_compliant_resources.append(web_acl["ARN"])
            else:
                raise e
```